node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/UnixPath.java

```diff
@@ -82,9 +84,19 @@ public String getPermissions() {
      * @param permissions Example: "rwxr-x---" means rwx for owner, rx for group,
      *                    and no permissions for others.
      */
-    public void setPermissions(String permissions) {
+    public UnixPath setPermissions(String permissions) {
```

Comment:
Update the other methods to also return `UnixPath`. F.ex. seems reasonable to chain `createDirectory()` with some of these methods.

Reply:
Done